02-technical-guides/hierarchies/main.ipynb

```diff
@@ -0,0 +1,2608 @@
+{
```

Comment:
<blockquote>and each hierarchy has it's own set of levels (starting with a 0-based index).</blockquote><p>has its own</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Fixed!